app/assets/css/utilities.css

```
@@ -1,3 +1,36 @@
+	font-weight: 600;
+	font-display: swap;
+}
+
+@font-face {
+	font-family: "Whitney";
+	src: url("https://cdn.skyra.pw/whitney-font/v2/Bold.woff") format("woff");
+	font-weight: 700;
+	font-display: swap;
+}
+
+@utility font-whitney {
```

Stylelint (17.10.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)